### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by falconshark**
Hello, I am a FFXIV player from Hong Kong, and very like your weather forecast tool as I always fishing in game.
I try to provide the Chinese version translation (In fact, the area and version name translation is from Chinese server...),
hope it is helpful.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -131,6 +131,12 @@ <h1 class="modal-title fs-5" id="advancedOptionsModalLabel">
         >
           日本語
         </button>
+        <button
+        class="btn btn-outline-secondary btn-sm"
```
Comment (Kikuflare) [inline index.html]: Lines 135 to 139 should have an additional two spaces of indentation.

**Review by Kikuflare — CHANGES_REQUESTED** [review]
Hi @falconshark ,

Thank you for contributing to this project. I've left a small comment regarding indentation in the `index.html` file, but everything else seems to look OK to me. Please make the requested change and I will merge your pull request.

File: index.html (hunk 1 above)
Comment (falconshark) [reply index.html]: Thank you very much !
I try to fix the indentation, currently it should be OK now.

**Comment by Kikuflare** [comment]
@falconshark 

Thank you for the quick update. I have confirmed the changes and merged the pull request.

Please add the official translations for the Dawntrail areas when the Chinese version of the game is updated. 😊

**Pull request merged**